salt/returners/etcd_return.py

```diff
@@ -132,7 +128,7 @@ def save_load(jid, load, minions=None):
     """
     Save the load to the specified jid
     """
-    log.debug("sdstack_etcd returner <save_load> called jid: {0}".format(jid))
+    log.debug("sdstack_etcd returner <save_load> called jid: {}".format(jid))
```

Comment:
```suggestion
    log.debug("sdstack_etcd returner <save_load> called jid: %s", jid)
```
If you're up to it, throughout the module...